docs/element-definitions.md

```diff
@@ -1,1419 +1,1623 @@
       ]
-    },
-    "title": {
-      "element": "string",
-      "content": "Polls API"
     }
   },
+  "content": [
+    {
+      "element": "string"
+    },
+    {
+      "element": "number"
+    }
+  ]
+}
+
+```
+
+Type Element matching only lists of one of `string`, `number`.
+
+```json
+
+{
+  "element": "array",
   "attributes": {
-    "metadata": {
+    "typeAttributes": {
       "element": "array",
       "content": [
         {
-          "element": "member",
-          "meta": {
-            "classes": {
-              "element": "array",
-              "content": [
-                {
-                  "element": "string",
-                  "content": "user"
-                }
-              ]
-            }
-          },
-          "content": {
-            "key": {
-              "element": "string",
-              "content": "HOST"
-            },
-            "value": {
-              "element": "string",
-              "content": "http://polls.apiblueprint.org/"
-            }
-          }
+          "element": "string",
+          "content": "fixedType"
         }
       ]
     }
   },
   "content": [
     {
-      "element": "category",
-      "meta": {
-        "classes": {
-          "element": "array",
-          "content": [
-            {
-              "element": "string",
-              "content": "resourceGroup"
-            }
-          ]
-        },
-        "title": {
-          "element": "string",
-          "content": "Question"
-        }
-      },
-      "content": [
-        {
-          "element": "copy",
-          "content": "Resources related to questions in the API."
-        }
-      ]
+      "element": "string"
+    },
+    {
+      "element": "number"
     }
   ]
 }
+
 ```
 
-### Copy (Base API Element)
+---
 
-Copy element represents a copy text—a textual information in API description.
-Its content is a string and it MAY include information about the media type
-of the copy's content.
+### Member Element
+Type with domain of all _properties_.
 
-Unless specified otherwise, a copy element's content represents the
-description of its parent element and SHOULD be used instead of parent
-element's description metadata.
+Quoting from the [Element section](#element):
+> A _property_ is a tuple (`key`, `value`) where
+> - `key` SHALL be a non-empty, finite character string
+> - `value` SHALL be an _Element_
+> - two properties SHALL be equal if their keys are.
 
-#### Properties
+#### Template
+- `element` - `"member"`
+- `attributes`
+  - `typeAttributes` ([Array](#array-element)[[String](#string-element)])
+    - `required` ([String](#string-element)) - Property MUST be present in Object containing it. I.e. restricts the domain of the containing Object Element type to one containing this property.
+    - `optional` ([String](#string-element)) - Property MAY NOT be present in Object containing it. I.e. expands the domain of the containing Object Element type to one not containing this property.</p>
```

Comment:
`present in value represented by the containing [Object Element](#object-element)`